src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java

```diff
@@ -259,9 +259,9 @@ private void updateRatingHint(int rating) {
   @SuppressWarnings("unchecked")
   private void plotRatingDistributions(List<RatingStat> ratingStats, Player player) {
     XYChart.Series<String, Integer> series = new XYChart.Series<>();
-    series.setName(i18n.get("ranked1v1.players"));
+    series.setName(i18n.get("ranked1v1.players", LeaderboardService.MINIMUM_GAMES_PLAYED_TO_BE_SHOWN ));
     series.getData().addAll(ratingStats.stream()
-        .sorted(Comparator.comparingInt(RatingStat::getCount))
+        .sorted(Comparator.comparingInt(RatingStat::getRating).reversed())
```

Comment:
I'd go without `.reversed()` so that low numbers are left and high numbers are right